TS3Client/Full/PacketHandler.cs

```diff
@@ -125,7 +125,7 @@ private void ConnectUdpClient(string host, ushort port)
 
 				remoteAddress = new IPEndPoint(ipAddr, port);
 
-				udpClient = new UdpClient();
+				udpClient = new UdpClient(remoteAddress.AddressFamily);
```

Comment:
Documentation says first parameter is the port a 32bit int and the AddressFamily is the second parameter.

Reply:
There are multiple constructors with one parameter:

- [UdpClient(Int32)](https://docs.microsoft.com/en-us/dotnet/api/system.net.sockets.udpclient.-ctor?view=netframework-4.7#System_Net_Sockets_UdpClient__ctor_System_Int32_)
- [UdpClient(IPEndPoint)](https://docs.microsoft.com/en-us/dotnet/api/system.net.sockets.udpclient.-ctor?view=netframework-4.7#System_Net_Sockets_UdpClient__ctor_System_Net_IPEndPoint_)
- [UdpClient(AddressFamily)](https://docs.microsoft.com/en-us/dotnet/api/system.net.sockets.udpclient.-ctor?view=netframework-4.7#System_Net_Sockets_UdpClient__ctor_System_Net_Sockets_AddressFamily_)

So that should work just fine.

Reply:
Ok, Well I dont have a v6 server to test on :(

Reply:
This is a good request, but unfortunately I already have added this on the develop branch: 03c42f6c79e9baa78ef7a3d1bc096ae412fc828b